servers/cu/src/domain/index.js

```diff
@@ -195,6 +195,18 @@ export const createApis = async (ctx) => {
     compute: fromPromise(() => _metrics.metrics())
   }
 
+  const evaluationCounter = MetricsClient.counterWith({})({
+    name: 'total_evaluations',
+    description: 'The total number of evaluations on a CU',
+    labelNames: ['processId', 'cron', 'dryRun', 'error']
+  })
+
+  const gasCounter = MetricsClient.counterWith({})({
+    name: 'total_gas_used',
+    description: 'The total number of gas used on a CU',
+    labelNames: ['processId', 'cron', 'dryRun', 'error']
+  })
+
```

Comment:
I thought you were just going to use `gauge` since it seemed that the labels weren't working on Counter?

Reply:
Figured out why counter wasn't working - if exemplars are enabled, accepts `{ value, labels, exemplars }` as arg. It is now implemented in this way.